.vscode/settings.json

```diff
@@ -10,7 +10,7 @@
         "editor.defaultFormatter": "ms-python.black-formatter",
         "editor.formatOnSave": true,
         "editor.codeActionsOnSave": {
-            "source.organizeImports": true
+            "source.organizeImports": "explicit"
```

Comment:
What impact does this have?

I want to make sure my pre-commit setup will agree with this

Reply:
"VSCode true is set to "explicit" since VSCode 1.85.0. It seems boolean values were supported until 1.84.0."

https://stackoverflow.com/questions/77637621/vscode-workspace-settings-change-on-its-own
https://code.visualstudio.com/updates/v1_85#_code-actions-on-save-and-auto